source/mir/appender.d

```diff
@@ -38,6 +38,48 @@ struct ScopedBuffer(T, size_t bytes = 4096)
         return *cast(inout(T[_bufferLength])*)&_scopeBufferPayload;
     }
 
+    ///
+    void reserve(size_t n) @trusted scope
```

Comment:
Can we just call `prepare` inside?

Reply:
We could but that's gonna waste cycles.

Reply:
Resolved via d13e80f.

Reply:
Commented them aswell at https://github.com/libmir/mir-algorithm/compare/14347b9e0a2493c32f7fad96e538b99248cb2df3..a0902dee6bffedbe27ad0b22c75e27818e6b5774.